app/controllers/v1/AdminDataController.scala

```diff
@@ -60,8 +60,8 @@ class AdminDataController @Inject() (repository: AdminDataRepository, val messag
         }
         case None => NotFound(errAsJson(NOT_FOUND, "Not Found", Messages("controller.not.found", v.id)))
       })).recover({
-      case _ => {
-        logger.error(s"Unable to get record from database")
+      case e => {
```

Comment:
I'm not sure if a specific exception is ever returned from the circuit breaker?

Reply:
Yep, I saw a scenario when running but couldn't see the stacktrace so added this to the logging.